tests/test_data_api_client.py

```diff
@@ -632,8 +632,8 @@ def test_email_address_with_valid_admin_domain_is_true(self, data_client, rmock)
         assert result is True
 
     def test_email_address_with_invalid_admin_domain_is_false(self, data_client, rmock):
-        rmock.get(
-            "http://baseurl/users/valid-admin-email?email_address=kev%40not-gov.uk",
+        rmock.post(
+            "http://baseurl/users/valid-admin-email",
             json={"valid": False},
             status_code=200
         )
```

Comment:
For POST requests, we can also assert the JSON from the last call (see line 558 for an example):

```
assert rmock.last_request.json() == {
    "emailAddress": True
}
```

Reply:
great suggestion @katstevens have added these assertions in 9b3ebff.

Reply:
Great 👍 (could we add for the `test_is_email_address_with_valid_buyer_domain_(true|false)` tests as well?)

Reply:
good spot, missed those. Added those now in 55c6c37.